starter_code/public/javascript/index.js

```diff
@@ -2,22 +2,57 @@ const charactersAPI = new APIHandler("http://ih-api.herokuapp.com")
 
 $(document).ready( () => {
   $('#fetch-all').on('click', (e) => {
-
+    var characters = [];
+    charactersAPI.getFullList().then(e => {
+      e.map((el, i) => {
+        var item = [
+          $("name").clone().append(' ' + el.name),
+          $("occupation").clone().append(' ' + el.occupation),
+          $("debt").clone().append(' ' + el.debt),
+          $("weapon").clone().append(' ' + el.weapon)
+        ]
```

Comment:
Cuenta la leyenda que el poderoso JQuery, sentado en su trono, no supo identificar los elementos que frente a él le puso @anaamilo . Fue entonces cuando apareció un poderoso TA y dijo: "No temáis gran JQuery. Esta mujer os quiere engañar. ¡Son clases!"

Y así fue como colgaron a @anaamilo por los tobillos por no mirar dos veces...